src/frontend/apps/impress/src/features/docs/doc-header/components/DocTitle.tsx

```diff
@@ -98,70 +90,30 @@ const DocTitleInput = ({ doc }: DocTitleProps) => {
     }
   };
 
-  const handleOnClick = () => {
-    if (isUntitled) {
-      setTitleDisplay('');
-    }
-  };
-
-  useEffect(() => {
-    if ((!debounceRef.current && !isUntitled) || !headingText) {
-      return;
-    }
-
-    setTitleDisplay(headingText);
-
-    if (debounceRef.current) {
-      clearTimeout(debounceRef.current);
-    }
-
-    debounceRef.current = setTimeout(() => {
-      handleTitleSubmit(headingText);
-      debounceRef.current = undefined;
-    }, 3000);
-  }, [isUntitled, handleTitleSubmit, headingText]);
-
   return (
     <>
       <Tooltip content={t('Rename')} placement="top">
         <Box
-          as="h2"
-          $radius="4px"
-          $padding={{ horizontal: 'tiny', vertical: '4px' }}
-          $margin="none"
-          contentEditable={isFirefox() ? 'true' : 'plaintext-only'}
-          onClick={handleOnClick}
-          onBlurCapture={(e) =>
-            handleTitleSubmit(e.currentTarget.textContent || '')
-          }
+          as="input"
+          defaultValue={isUntitled ? '' : titleDisplay}
           onKeyDownCapture={handleKeyDown}
-          suppressContentEditableWarning={true}
-          $color={
-            isUntitled
-              ? colorsTokens()['greyscale-200']
-              : colorsTokens()['greyscale-text']
-          }
-          $css={`
-            ${isUntitled && 'font-style: italic;'}
-            cursor: text;
-            font-size: ${isMobile ? '1.2rem' : '1.5rem'};
-            transition: box-shadow 0.5s, border-color 0.5s;
-            border: 1px dashed transparent;
-            
-            &:hover {
-              border-color: rgba(0, 123, 255, 0.25);
-              border-style: dashed;
-            }
-
-            &:focus {
-              outline: none;
-              border-color: transparent;
-              box-shadow: 0 0 0 2px rgba(0, 123, 255, 0.25);
-            }
+          placeholder={untitledDocument}
+          aria-label={t('doc title input')}
+          name="doc-title"
+          $radius="4px"
+          onBlurCapture={(event) => handleTitleSubmit(event.target.value)}
+          $color={colorsTokens()['greyscale-text']}
+          $flex={1}
+          $width="100%"
```

Comment:
Can we put the field the size of the text ? You will have the `Rename` tag correctly placed like that:
![image](https://github.com/user-attachments/assets/ef2926b4-fb43-4bf2-bf30-829fd25f912c)